### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by archived-aft-bogus-german-visa-rejectio** [langfuse/langfuse#941] [Ideas]
thanks!

**Comment by khareyash05** [comment]
Hey Can I work on it @archywillhe ?

**Comment by marcklingen** [comment]
Totally get why this would be cool! As most of the application uses shadcn/ui adding dark mode should be fairly simple. The only thing to investigate is how the charts on the dashboard are affected (using tremor). @khareyash05 are you interested in contributing this and doing some research on it first to understand the impact across the application?

**Reply by khareyash05** [reply]
Yes @marcklingen . I would like to contribute. Charts would be a concern. Will dive deep into it. Thanks

**Reply by khareyash05** [reply]
Tremor Charts are easy to dive in using Tailwind classes. Read through their docs here [https://www.tremor.so/docs/getting-started/theming](https://www.tremor.so/docs/getting-started/theming)

**Comment by david1542** [comment]
Voting for this suggestion :)

**Comment by khareyash05** [comment]
https://github.com/langfuse/langfuse/pull/968. Linked PR.

**Comment by RichardKruemmel** [comment]
Exciting news!

Dark mode is here! 🎉 
Check out the details in our changelog [here](https://langfuse.com/changelog/2024-05-23-dark-mode). 

Enjoy the dark side! 🌑

**Reply by marcklingen** [reply]
Wuhuuuu
